### PR TITLE
Updates Euclidean Distance formula

### PR DESCRIPTION
**Pull request by Wabbrik**
### Description

Fixes wrong euclidean distance formula.

### Motivation

Noticed the issue while reading the docs.

### Additional details

N/A

### Related issues and pull requests

N/A

### CODE REVIEW

**Comment by yarusome** [comment]
`<annotation>` is not displayed but only generated by [MathJax](https://www.mathjax.org) for a11y. To correct the formula, the following can be used:

```html
<math><semantics><mrow><msqrt><msup><mrow><mo stretchy="false">(</mo><msub><mi>A</mi><mi>x</mi></msub><mo>−</mo><msub><mi>B</mi><mi>x</mi></msub><mo stretchy="false">)</mo></mrow><mn>2</mn></msup><mo>+</mo><msup><mrow><mo stretchy="false">(</mo><msub><mi>A</mi><mi>y</mi></msub><mo>−</mo><msub><mi>B</mi><mi>y</mi></msub><mo stretchy="false">)</mo></mrow><mn>2</mn></msup><mo>+</mo><msup><mrow><mo stretchy="false">(</mo><msub><mi>A</mi><mi>z</mi></msub><mo>−</mo><msub><mi>B</mi><mi>z</mi></msub><mo stretchy="false">)</mo></mrow><mn>2</mn></msup></msqrt></mrow><annotation>\sqrt{ {( A_x - B_x )}^2 + {( A_y - B_y )}^2 + {( A_z - B_z )}^2 }</annotation></semantics></math>
```

**Comment by Wabbrik** [comment]
> `<annotation>` is not displayed but only generated by [MathJax](https://www.mathjax.org) for a11y. To correct the formula, the following can be used:
> 
> ```
> <math><semantics><mrow><msqrt><msup><mrow><mo stretchy="false">(</mo><msub><mi>A</mi><mi>x</mi></msub><mo>−</mo><msub><mi>B</mi><mi>x</mi></msub><mo stretchy="false">)</mo></mrow><mn>2</mn></msup><mo>+</mo><msup><mrow><mo stretchy="false">(</mo><msub><mi>A</mi><mi>y</mi></msub><mo>−</mo><msub><mi>B</mi><mi>y</mi></msub><mo stretchy="false">)</mo></mrow><mn>2</mn></msup><mo>+</mo><msup><mrow><mo stretchy="false">(</mo><msub><mi>A</mi><mi>z</mi></msub><mo>−</mo><msub><mi>B</mi><mi>z</mi></msub><mo stretchy="false">)</mo></mrow><mn>2</mn></msup></msqrt></mrow><annotation>\sqrt{ {( A_x - B_x )}^2 + {( A_y - B_y )}^2 + {( A_z - B_z )}^2 }</annotation></semantics></math>
> ```

You are right! Will fix in this commit if it's ok!

File: files/en-us/games/techniques/3d_collision_detection/index.md [hunk 1]
```diff
@@ -77,7 +77,7 @@ To check whether a sphere contains a point we need to calculate the distance bet
...
+Taking into account that the Euclidean distance between two points _A_ and _B_ is <math><semantics><mrow><msqrt><msup><mrow><mo stretchy="false">(</mo><msub><mi>A</mi><mi>x</mi></msub><mo>−</mo><msub><mi>B</mi><mi>x</mi></msub><mo stretchy="false">)</mo></mrow><mn>2</mn></msup><mo>+</mo><msup><mrow><mo stretchy="false">(</mo><msub><mi>A</mi><mi>y</mi></msub><mo>−</mo><msub><mi>B</mi><mi>y</mi></msub><mo stretchy="false">)</mo></mrow><mn>2</mn></msup><mo>+</mo><msup><mrow><mo stretchy="false">(</mo><msub><mi>A</mi><mi>z</mi></msub><mo>−</mo><msub><mi>B</mi><mi>z</mi></msub><mo stretchy="false">)</mo></mrow><mn>2</mn></msup></msqrt></mrow><annotation>\sqrt{ {( A_x - B_x )}^2 + {( A_y - B_y )}^2 + {( A_z - B_z )}^2 }</annotation></semantics></math>, our formula for point vs. sphere collision detection would work out like so:
```
Comment (Josh-Cena) [inline files/en-us/games/techniques/3d_collision_detection/index.md:80]:
```suggestion
Taking into account that the Euclidean distance between two points _A_ and _B_ is <math><semantics><msqrt><mrow><mo stretchy="false">(</mo><msub><mi>A</mi><mi>x</mi></msub><mo>−</mo><msub><mi>B</mi><mi>x</mi></msub><msup><mo stretchy="false">)</mo><mn>2</mn></msup><mo>+</mo><mo stretchy="false">(</mo><msub><mi>A</mi><mi>y</mi></msub><mo>−</mo><msub><mi>B</mi><mi>y</mi></msub><msup><mo stretchy="false">)</mo><mn>2</mn></msup><mo>+</mo><mo stretchy="false">(</mo><msub><mi>A</mi><mi>z</mi></msub><mo>−</mo><msub><mi>B</mi><mi>z</mi></msub><msup><mo stretchy="false">)</mo><mn>2</mn></msup></mrow></msqrt><annotation encoding="TeX">\sqrt{(A_x - B_x)^2 + (A_y - B_y)^2 + (A_z - B_z)^2}</annotation></semantics></math>, our formula for point vs. sphere collision detection would work out like so:
```

**Review by Josh-Cena — APPROVED** [review]
Thank you!

**Pull request merged**